runtime/doc/treesitter.txt

```diff
@@ -630,75 +631,75 @@ get_node({opts})                                   *vim.treesitter.get_node()*
                   true)
 
     Return: ~
-        |TSNode| | nil Node at the given position
+        (`TSNode?`) Node at the given position
 
 get_node_range({node_or_range})              *vim.treesitter.get_node_range()*
     Returns the node's range or an unpacked range table
 
     Parameters: ~
-      • {node_or_range}  (|TSNode| | table) Node or table of positions
+      • {node_or_range}  (`TSNode|table`) Node or table of positions
```

Comment:
💯 Working around those pipe conflicts was annoying.

Reply:
I forgot about that. I guess that's another reason to not do the tagging since it conflict too much with unions.

Reply:
Yeah. The loss of help links is a blow for treesitter specifically (I don't think we use much userdata elsewhere?), but that help needs more love anyway. 

We can always tack a simple `|TSNode|` at the end of the annotation comment (if there's not a link elsewhere in the docs already).